packages/wethegit-components/src/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react"
+
+import styles from "./accordion.module.scss"
```

Comment:
The styles for the component itself should be imported last to help with CSS loading order.